docs/checking-broken-links.md

```diff
@@ -54,21 +54,22 @@ worktree: /var/www/owncloud/docs | component: server | version: master
 If you already installed the Antora dependencies via `yarn install` you already got a broken link checker included, you can simple execute the following command:
 
 ```console
-yarn linkcheck http://localhost:8080
+yarn linkcheck http://localhost:8080 | grep "BROKEN"
 ```
 
 #### Example Output
 
 ```console
 ...
-$ broken-link-checker --filter-level 3 --recursive --verbose https://doc.owncloud.com
-Getting links from: http://localhost:8080/client/automatic_updater.html
-├───OK─── https://github.com/owncloud/client/edit/master-antora/docs/modules/ROOT/pages/automatic_updater.adoc
-├───OK─── https://doc.owncloud.org/branded_clients/
-├─BROKEN─ https://owncloud.org/history/ (HTTP_404)
-Finished! 60 links found. 57 excluded. 1 broken.
+├─BROKEN─ http://mechanics.flite.com/blog/2014/07/29/using-innodb-large-prefix-to-avoid-error-1071/ (BLC_UNKNOWN)
+├─BROKEN─ http://bucket.hostname.domain/ (ERRNO_ENOTFOUND)
+├─BROKEN─ http://hostname.domain/bucket (ERRNO_ENOTFOUND)
+├─BROKEN─ https://example.com/owncloud (HTTP_404)
+├─BROKEN─ https://example.com/owncloud (HTTP_404)
 ...
 ```
+Please consider that only the first entry is a broken link as the others in the list
```

Comment:
Maybe:
```
Note: only the first entry is a real broken link. The others in the list are example links and therefore not broken.
```
@settermjd how would you write this?

Reply:
@phil-davis info, this is page markdown (.md), not asciidoc (.adoc). md does not have the directive NOTE: and it will not render like we are used in adoc. The only thing we can do is write Note: and some text which will all be all rendered as text. I am open for any improvement.

Reply:
@phil-davis, good question. I'll have to review and think it over.

Reply:
tbh, I'm not sure what the comment means.

Reply:
Please clarify what you mean, @mmattel.

Reply:
@mmattel, given your further clarification, I suggest the following sentence "_Note: in the example output above, only the first entry is a genuine broken link. All the others in the list are example links, and therefore not broken._".

Reply:
Done